content/en/docs/pipelines/installation/overview.md

```diff
@@ -131,21 +151,27 @@ Interfaces
   * Kubeflow Pipelines UI via the **Open Pipelines Dashboard** link in the 
     GCP Console
   * Kubeflow Pipelines SDK in Cloud Notebooks
+  * Kubeflow Pipelines endpoint of your instance is auto-configured for you.
```

Comment:
The use of full stops in bullet points is not consistent, so I suggest to remove the dot here.